### PR TITLE
Add -A option to ps command to list all Fire TV processes

### PR DESCRIPTION
**Pull request by mano3m**
Fixes [#102565](https://github.com/home-assistant/core/issues/102565) and [https://github.com/JeffLIrion/python-androidtv/issues/292](https://github.com/JeffLIrion/python-androidtv/issues/292) and updates [https://github.com/JeffLIrion/python-androidtv/commit/e8b09aab7931a65be7e9beb71409c079da172b43](https://github.com/JeffLIrion/python-androidtv/commit/e8b09aab7931a65be7e9beb71409c079da172b43)

### CODE REVIEW

File: androidtv/constants.py [hunk 1]
```diff
@@ -209,7 +209,7 @@ class DeviceEnum(IntEnum):
 CMD_RUNNING_APPS_ANDROIDTV = "ps -A | grep u0_a"
 
 #: Get the running apps for a Fire TV device
-CMD_RUNNING_APPS_FIRETV = "ps | grep u0_a"
+CMD_RUNNING_APPS_FIRETV = "ps -A | grep u0_a"
```
Comment (JeffLIrion) [inline androidtv/constants.py:212]: A unit test is failing because this constant is the same as `CMD_RUNNING_APPS_ANDROIDTV`.

Comment (mano3m) [reply androidtv/constants.py:212]: You want me to remove the FIRETV constant and refer to the ANDROIDTV one from everywhere?

Comment (JeffLIrion) [reply androidtv/constants.py:212]: Please consolidate the two into a single `CMD_RUNNING_APPS`.

Comment (mano3m) [reply androidtv/constants.py]: Ok done 👍

**Comment by mano3m** [comment]
Di you need more to merge this?

**Pull request merged**